src/Translation/Translator.php

```diff
@@ -77,15 +50,6 @@ public function loadCurrentLocale(): string
         return empty($preferredLanguage) ? $this->defaultLocale : $preferredLanguage;
     }
 
-    /**
-     * {@inheritdoc}
-     * @codeCoverageIgnore
-     */
-    public function setLocale($locale = 'en'): void
```

Comment:
@SpartakusMd why did you decide to remove this method? It can be used for manually creating translation.